src/java/org/apache/cassandra/repair/RepairRunnable.java

```diff
@@ -363,27 +382,57 @@ private NeighborsAndRanges getNeighborsAndRanges()
         return new NeighborsAndRanges(shouldExcludeDeadParticipants, allNeighbors, commonRanges);
     }
 
+    @VisibleForTesting
+    public static NeighborsAndRanges createNeighbordAndRangesFromOptions(RepairOption options)
+    {
+        Preconditions.checkArgument(!options.getHosts().isEmpty(), "There should be at least 1 host when validateNeighbors is set to true");
```

Comment:
`validateNeighbors` is still here. 
Perhaps I wasn't clear, so let me expand.
`validateNeighbors` cannot be set to `true` or `false` because the only place where it appears is assertion messages.

It probably should be changed to reflect service being offline or not, e.g.:
`There should be at least 1 host when repairing via offline service`

Reply:
forgot to change the messages.. fixed